### ISSUE TITLE
Nokia PC Suite reads very poorly

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#349]
**Reported by timeless on 2009-07-02 12:58**
Windows 7 RC1
Nokia PC Suite 7.1.26.0:
http://nds1.nokia.com/files/support/global/phones/software/Nokia_PC_Suite_7_1_26_0_eng_us_web.exe
NVDA 0.6p3.2

this could be entirely Nokia's fault, please feel free to complain to them.

I hope you don't mind, it's easier to have a tracker which has the baseline and then individual items for specific problems.
**Blocked by #350, #351**

**Comment by leonardder** [comment]
Coming back to @josephsl's last comment in here, PC Suite is hardly used nowadays. I vote for a won't fix

**Comment by jcsteh** [comment]
Agreed. Closing as won't fix.
